dashcam/package/object-detection/files/detect_hdc.py

```diff
@@ -79,7 +79,14 @@ def combine_images(images, grid_size, model_size):
 
     return combined_img, orig_images
 
-def transform_box(box, model_size, grid_size, index):
+CLASS_NAMES = ['face', 'person', 'license-plate', 'car', 'bus', 'truck', 'motorcycle', 'bicycle']
+
+def get_label(class_id):
```

Comment:
Might be good to use type hints in new python code

```
def get_label(class_id: int) -> Optional[str]:
```

It makes the code easier to read and there are vscode plugins that do type checking on annotated code

No need to do this in this PR if it would slow things down